sessions/poster-making/README.md

```diff
@@ -4,34 +4,51 @@
+- **Objective**: Get people to learn some part of the code base in depth and through that process of learning, create materials that can be used by others to learn that part of the code base. Promote collaboration, knowledge transfer through deep dives and generation of materials that can be used by others in the future.
+- **Activity**: These sessions consist on gathering people into small groups (2~4) to explore and understand in depth one of the selected topics. At the end of the session, everyone should have a poster ready (this will be fun!). At the end of each day, all of the poster groups will give a one-minute presentation of their poster.
 - **Outcome**: The posters and discussions will then be converted into docs and/or specs to the IPFS repos.
 
 ## Details
 
-We should do this multiple times during the meeting, with multiple small groups working in parallel.
+During this session, use your time to learn some part of the code base in depth together with a small team of collaborators. Together, create a poster that others can use to learn about that part of the code base.
+
+Some of the things to do during this session together with the other people on your poster team:
+
+- **Review any formal documentation** relevant to the topic you've chosen
+- **Find and read all the "pseudo-specs"** -- gather and read through all the partial documentation that's floating around. Discuss it together.
+- **Look through the related code**
+- **Look for relevant reference materials**: papers, recordings of talks, etc.
+- **Clarify and test your understanding** by discussing all of this information  
+- **Interact with experts in the room** - people like Tech Leads will be circulating around the groups
+- **Compile all of this information** into something that's easy to consume (a poster)
+- **Plan how this information can be incorporated** into the existing documentation and repositories.
+- **Choose who will present your poster.**
+- _If you have time, submit PR your new info into the corresponding repository._
+
+At the end of the day, each group will have 1 minute to present their poster. Make sure to choose in advance who will be presenting.
+
+## How you will choose which poster to work on
 
-These topics will be pre selected (pre proposed) and they should be things that people keep asking for better explanations or simply are things that are "complex" and we need to document better. Think things like:
+Poster teams should have a maximum of 4 people and a minimum of 2. People will choose which poster to work on using this process:
+- Leading up to the session, anyone can propose poster topics using the [instructions below](#proposed-poster-making-sessions)
+- The meeting organizers will draw a spot on the ground for each of the posters.
```

Comment:
👍

Reply:
GIven that we are doing this explanation during lunch. Would it be better to have "spots" with "tickets" so that people can run, pick one and finish their meal? @flyingzumwalt